core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ProfileClassWarehouse.scala

```diff
@@ -878,6 +948,141 @@ case class SQLTaskAggMetricsProfileResult(
   }
 }
 
+case class IODiagnosticResult(
```

Comment:
Can we have a comment on the case class that tells which output it is used for, and then for some of the fields; especially `metrics`, it will be ice to tell more details about them, and the timeUnit if it is not part of the name.

Reply:
thanks, updated.